### PR TITLE
Button for a flow layouting was added. Button is hidden if onFlowLayo…

### PR DESCRIPTION
**Pull request by Ladone3**
…ut function isn't specified.

### CODE REVIEW

File: src/ontodia/workspace/workspace.ts [hunk 1]
```diff
@@ -127,6 +132,10 @@ export class Workspace extends Component<WorkspaceProps, State> {
         } as MarkupProps & React.ClassAttributes<WorkspaceMarkup>);
     }
 
+    protected getFlowLayoutHandler = (): (() => void) => {
```
Comment (AlexeyMz) [inline src/ontodia/workspace/workspace.ts]: Use a usual function definition here, e.g.:
```
protected getFlowLayoutHandler(): () => void {
  // ...
}
```

File: styles/widgets/_toolbar.scss [hunk 2]
```diff
@@ -21,3 +24,17 @@
 .ontodia-toolbar .ontodia-toolbar__redo {
   display: none;
 }
+
+.ontodia-toolbar .ontodia-toolbar__layout-group {
```
Comment (AlexeyMz) [inline styles/widgets/_toolbar.scss]: Use Sass nested selectors instead:
```
.ontodia-toolbar {
  ...
  &__layout-group { ... }
}
```

File: styles/widgets/_toolbar.scss [hunk 3]
```diff
@@ -21,3 +24,17 @@
...
+  float: left;
+
+  > label {
+    margin-right: 5px;
+  }
+
+  .btn-group, button {
+    height: 100%;
+  }
+}
```
Comment (AlexeyMz) [inline styles/widgets/_toolbar.scss]: Missing end-of-file newline here.

File: src/ontodia/workspace/workspace.ts (hunk 1 above)
Comment (Ladone3) [reply src/ontodia/workspace/workspace.ts]: In that case we wont be able to use 'this' in overridden function, or we will need to bind this function. Is it OK?

**Pull request merged**